source/kann-metadata.json

```diff
@@ -0,0 +1,207 @@
+[
+    {
+        "name": "Transpose",
+        "category": "Transform"
+    },
+    {
+        "name": "Substraction",
+        "category": "Transform"
+    },
+    {
+        "name": "LayerNorm",
+        "category": "Normalization"
+    },
+    {
+        "name": "Output",
+        "category": "Data"
+    },
+    {
+        "name": "MaxWithConstant",
+        "category": "Transform"
+    },
+    {
+        "name": "FullMapping",
+        "category": "Transform"
+    },
```

Comment:
this is an abstract class, it will never be serialized in a model

Reply:
I added a filter when generating kann-metadata.json to not consider abstract classes